### DISCUSSION TITLE
"Strain relief" cutout in wand cage?

### DISCUSSION THREAD

**Discussion by nomakewan** [gpstar81/GPStar-proton-pack#306] [Ideas]
As people are starting to get their 49-LED barrel parts in, I've noticed a common complaint is that the wires are thicker than the Hasbro wires. This means you cannot loop them into the Hasbro strain relief (which was already a super tight fit even for the stock wires, to be fair). The wires are also longer, making it easier for them to get caught by the barrel pop mechanism. Perhaps it might be worth it to include cutouts in the "front" of the cage (the side that will face the front of the wand) that users can bend the wire through so that it stays fixed in place more easily? A rudimentary strain relief., so that rather than the whole wire moving around inside the wand, only the part between the cage and the side of the barrel is moving.

As a reminder, the strain reliefs for the barrel wiring on the third-generation wands works like this:

![IMG_6946s](https://github.com/gpstar81/GPStar-proton-pack/assets/11485516/79f9901c-5501-432c-b935-9af88eebcfaf)

**Comment by Toy203** [comment]
Just got the wires in, doesn't seems they get particularly stressed by the movement. I kept them behind the top right wide  leg of the cage ( top right position from your pic) and they move freely from what I saw so far.
No movement on top

**Comment by Toy203** [comment]
Update on this, i've also replaced the vent light with Frutto's. Barrel cables are still moving freely behind the cage without much issue and i don't see any tangling.
